### ISSUE TITLE
Don't use WP_CLI::error() when installing translations for multiple locales

### ISSUE THREAD

**Issue by ocean90** [wp-cli/language-command#61]
This is about https://github.com/wp-cli/language-command/blob/f71622d223226e1d738f1f1a645c47d73cb921e5/src/Plugin_Language_Command.php#L212-L225

Example command: `wp language plugin install <plugin-name> de_DE de_CH`
When the installation of `de_DE` fails the command should continue to install `de_CH`. The failed installation should be documented as warning.
This would be similar to the media regenerate command which uses `Utils\report_batch_operation_results()`.

**Comment by schlessera** [comment]
Yes, seems that this would be more consistent to what other similar commands are doing.

Are you up for providing a PR for this?

**Comment by ocean90** [comment]
Started in https://github.com/wp-cli/language-command/pull/63.

**Comment by ocean90** [comment]
If #63 looks good we should probably do the same for the uninstall commands.
